Documentation/MRTKNuGetPackage.md

```diff
@@ -0,0 +1,66 @@
+# Mixed Reality Toolkit NuGet Package
+
+Mixed Reality Toolkit (MRTK) is now available as a NuGet package on NuGet.org. There are some differences when it comes to consuming NuGet version of MRTK as opposed to a .unitypackage, read [NuGet Package Considerations](###%20NuGet%20Package%20Considerations) below. If any issues are encountered, file an issue using this [template](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/new?assignees=&labels=Bug,Package%20Management%20-%20NuGet&template=bug-report.md&title=).
+
+## Installing the NuGet Package
+
+Follow these instructions to add the Mixed Reality Toolkit as a NuGet package to your project.
+
+1. Download the latest [NuGetForUnity](https://github.com/GlitchEnzo/NuGetForUnity/releases/latest) .unitypackage
+1. Import the package into your Unity project, [instructions](https://docs.unity3d.com/Manual/AssetPackages.html).
+1. Configure NuGet For Unity, click on the menu **Edit > Preferences...**.
+    - On the left, find **NuGet for Unity** tab.
+    - Un-check **Read-Only Package Files**.
+1. In the Unity menu bar, click on **NuGet > Manage NuGet Packages**.
+1. In the Search box, enter `Microsoft.MixedReality.Toolkit`, and choose the package you would like to install.
```

Comment:
this could also benefit from screen shot

Reply:
when adding screen shot, it is probably worth also calling out the nuget for unity version used in a comment so that there isn't confusion if ui doesn't match exactly